.github/CONTRIBUTING.md

```diff
@@ -7,6 +7,14 @@
 Search [the issue tracker][issue tracker] to ensure that there is no open issue addressing the bug.
 If that is the case, [open a new issue][new issue]. Make sure to include a clear title, description, and as much relevant information as possible.
 
+For feature requests, refer  [here](../docs/template-featureRequest.md)
+
+For bug reports, refer [here](../docs/template-bugReport.md)
+
+For new contributor introduction, refer [here](../docs/template-contributorIntro.md)
+
+For help requests, refer  [here](../docs/template-helpRequest.md)
```

Comment:
Double whitespace (between `refer` and `[here]`) here and in "feature requests".

Also, can you close all four of the above sentences with a period?